Lpd 56243

briangreenwald opened the pull request.

JhossephG committed:
LPD-44846 feature: create columns reviewDate and displayDate on ObjectEntry and an upgrade process to include them

JhossephG committed:
LPD-44846 test: integration tests reviewDate

izaera committed:
LPD-55605 Migrate CannotConfigureWorkflowSiteLevel test to Playwright

ling-alan-huang committed:
LPD-44271 SF

ling-alan-huang committed:
LPD-44271 SF

noornajj committed:
LPD-54628 Persist Asset Types and make it compatible with object definitions

noornajj committed:
LPD-54628 Add asset type confirmation modal

noornajj committed:
LPD-54628 Add empty input validation

noornajj committed:
LPD-54628 Add playwright tests

marcogalluzzi committed:
LPD-54424 Add new fragment renderer in the Folder display page template

marcogalluzzi committed:
LPD-54424 Add folder props transformer

marcogalluzzi committed:
LPD-54424 The title of a folder must be navigable

rebsilva committed:
LPD-55897 Allow SeparatorFields components to be used in Objects modules

noornajj committed:
LPD-54628 Persist selected values when switching between tabs

JhossephG committed:
LPD-44846 feature: map reviewDate, expiration and publishDate as system object fields

JhossephG committed:
LPD-44846 feature: service layer methods implementation

adolfopa committed:
LPD-54424 Add dropdown action to navigate to folder

izaera committed:
LPD-55605 Fix FF checks not taking company into account

izaera committed:
LPD-55605 Remove unused field

izaera committed:
LPD-55606 Introduce the goto/waitFor pattern in POM classes

Basically we want to have a waitFor() method in all POM classes that
guarantees page interactivity (ie: full load) on return.

Additionally, the goto() method that opens the page will invoke waitFor()
on exit to guarantee that the page is functional.

This should prevent flakyness.

izaera committed:
LPD-55605 Add missing awaits

izaera committed:
LPD-55605 Migrate failing Poshi tests to Playwright

adolfopa committed:
LPD-54424 Move edit folder action to base class

veroglez committed:
LPD-55891 Trigger an event when the editor is ready

izaera committed:
LPD-55605 Refactor tests and promote the waitFor/goto pattern to top level

veroglez committed:
LPD-55891 Adapt code to load CKEditor 5

veroglez committed:
LPD-55891 Adapt read-only mode

veroglez committed:
LPD-55891 Adapt the update of the language direction

veroglez committed:
LPD-55891 Adapt the change event

izaera committed:
LPD-55605 Centralize common POM logic and group sample tests

izaera committed:
LPD-55605 Fix more issues

veroglez committed:
LPD-55891 Update the lang for the editor

veroglez committed:
LPD-55891 Simplify the markup for the CKEditor 4

izaera committed:
LPD-55605 Simplifly page interactions and put asserts back to tests

Also remove methods to create CX from ClientExtensionsPage since that page
doesn't really allow that. EditXXXPage must be used instead.

marcogalluzzi committed:
LPD-54424 editFolder action is no longer needed from props transformers

marcogalluzzi committed:
LPD-54424 Remove no longer used additionalProps parameter

marcogalluzzi committed:
LPD-54424 Use objectEntryFolder from base class

gislayne-vitorino committed:
LPD-56165 Fix DDMTemplateVersion math in ConstraintResolver

izaera committed:
LPD-55605 Fix failing and flaky tests after refactor

izaera committed:
LPD-55605 Remove invalid test

I have investigated this and have found that we never required the Source
Code URL to be a valid URL.

Doing this for all types would require a considerable change in the CET
framework and I can't see any benefit other than breaking customer
installations since the source code URL field is purely informative so it's
up to the customer to put a valid URL or maybe a local path or anything
that can help them to review their stuff.

In any case DXP won't use that URL for anything.

gislayne-vitorino committed:
LPD-56165 Add missing language key

gislayne-vitorino committed:
LPD-56165 buildLang

veroglez committed:
LPD-55891 Adapt tests for CKEditor 5

marcogalluzzi committed:
LPD-54424 Add card view for folders

rtienpal committed:
LPD-56156 Create Expire Field using Schedule Field Component

veroglez committed:
LPD-55891 Add layoutMode param to RenderEditorTag

veroglez committed:
LPD-55891 Move all the markup to the taglib

veroglez committed:
LPD-55891 Add disabled param to EditorTag

veroglez committed:
LPD-55891 Add disabled prop to CKEditor 5 ClassicEditor

veroglez committed:
LPD-55891 Use visibility:hidden instead of opacity:0 in the label

veroglez committed:
LPD-55891 Take into account the CKEditor 5 in the edit mode

veroglez committed:
LPD-55891 Add input label for CKEditor 5

rtienpal committed:
LPD-56157 Add unit tests

veroglez committed:
LPD-55891 Remove unused typing

marcogalluzzi committed:
LPD-54424 Adding new custom renderer for action links

izaera committed:
LPD-55605 Activate FDS cell renderer FF in tests

marcogalluzzi committed:
LPD-54424 Using new custom renderer

marcogalluzzi committed:
LPD-54424 Fix - Adding download link for files in folders

izaera committed:
LPD-55605 Fix global JS tests

izaera committed:
LPD-55605 Fix FDS cell renderer tests

welitonpe committed:
LPD-54420 Updated Administrator Dashboard Router

welitonpe committed:
LPD-54420 Applying filters on the applications page

RoselaineMarquesMontero committed:
LPD-53494 Enable PlacedOrder API be filtered by orderStatusInfo

dsitu committed:
LPD-56462 Refactor CPCommerceOrders#CanCancelOrder from poshi to playwright

kevenleone committed:
LPD-54420 Make the category editable if value is empty

kevenleone committed:
LPD-54420 Follow design

izaera committed:
LPD-55605 SF

veroglez committed:
LPD-55891 Adapt to jakarta

sandrodw3 committed:
LPD-56489 Export

sandrodw3 committed:
LPD-56489 Add ExperienceCustomizerModal

sandrodw3 committed:
LPD-56489 buildLang

gislayne-vitorino committed:
LPD-56165 Add test to ensure behavior

rebsilva committed:
LPD-55897 Add friendlyURLSeparator prop to the ObjectDefinition interface

rebsilva committed:
LPD-55897 Create errors util and update ObjectActionContainer

rebsilva committed:
LPD-55897 Update SeparatorFields component
- add handleChange property to allow using the value outside the component if necessary
- add handleOnBlur property to allow using the onBlur event if necessary
- add hideReset property to make the reset button optional
- add warnings and help text properties
- use FieldFeedback component

rebsilva committed:
LPD-55897 Create SeparatorContainer component

rebsilva committed:
LPD-55897 Create AllowFriendlyURLContainer component

rebsilva committed:
LPD-55897 Update SeoContainer component

rebsilva committed:
LPD-55897 Update the usages of SeoComponent in Object Admin
- update the way we get and show the errors
- put the condition to render seo section outside the component instead of only making the URL checkbox disabled

rebsilva committed:
LPD-55897 Update the usages of SeoComponent in Model Builder
- update the way we get and show the errors
- put the condition to render seo section outside the component instead of only making the URL checkbox disabled

rebsilva committed:
LPD-55897 Add language keys

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

thiagobuarqque committed:
LPD-56572 warn developers when they are not using the fully qualified class name as the configuration id

JhossephG committed:
LPD-44846 feature: create schedule fields in DTO

welitonpe committed:
LPD-54420 Updated AdministratorSummary and removed loading from KPI charts

welitonpe committed:
LPD-54420 Created a custom hook to get product KPIdata

welitonpe committed:
LPD-54420 Adjuted filters on Apps, Solutions, and OrderPage

welitonpe committed:
LPD-54420 Fixed upload file

welitonpe committed:
LPD-54420 Update KPI and metrics component and hooks

JhossephG committed:
LPD-44846 feature: adjusting code for properly usage

Commit:
LPD-56590 reenable indexing for workflow instance

vamshikrishnamaddi committed:
LPD-55865 Fix Articles Count Logic

sandrodw3 committed:
LPD-56489 Calculate in frontend also for MarketplaceButton

sandrodw3 committed:
LPD-56489 Adapt tests

victorg1991 committed:
LPD-56487 Remove back layout since we we don't need it

victorg1991 committed:
LPD-56487 Don't render redirect input if it the value is empty

victorg1991 committed:
LPD-56487 Send redirect in the publish button to use the backURL

JhossephG committed:
LPD-44846 generated: buildService

JhossephG committed:
LPD-44846 generated: buildRest

JhossephG committed:
LPD-44846 sf

gabrielwas committed:
LPD-44846 Rename publishDate to displayDate in upgrade process

smottal committed:
LPD-55587 Service layer

smottal committed:
LPD-55587 Integration test

smottal committed:
LPD-55587 Update usage

smottal committed:
LPD-55587 Autogenerated Code (build-service-portal)

gabrielwas committed:
LPD-44846 Add versioning columns

noornajj committed:
Revert "LPD-53279 Persist selected values when switching between tabs"

This reverts commit bf9bf86c96d4d7d256296eff8a9eb8e24f233020.

noornajj committed:
Revert "LPD-53279 Add playwright tests"

This reverts commit 962629f1c6b70cb2909f7d057bbaa52607020219.

gabrielwas committed:
LPD-44846 buildService

noornajj committed:
Revert "LPD-53279 Add empty input validation"

This reverts commit d5d1b9b7b6cb5f5e1dd1a0b1ce20fd765c945751.

noornajj committed:
Revert "LPD-53279 Add asset type confirmation modal"

This reverts commit b1321d064346b4f99b72a935c47e697ca43e6c6b.

noornajj committed:
Revert "LPD-53279 Persist Asset Types and make it compatible with object definitions"

This reverts commit baceb6f04899a31086fd2311736334e60e7fafc4.

gabrielwas committed:
LPD-44846 Add upgrade step to ObjectEntryVersion new columns

gabrielwas committed:
LPD-44846 Populate object entry version display and review dates

gabrielwas committed:
LPD-44846 Replace checks by _getAttribute

marcogalluzzi committed:
LPD-54424 Fix import

marcogalluzzi committed:
LPD-54424 Improve typescript

gabrielwas committed:
LPD-44846 Persist displayDate and expirationDate

marcogalluzzi committed:
LPD-54424 Rename action ID

gabrielwas committed:
LPD-44846 Simplify code

marcoscv-work committed:
LPD-43525 - Add CMS Theme

rebsilva committed:
LPD-55897 Make right sidebar wider when showing object details

gabrielwas committed:
LPD-44846 Sort

marcoscv-work committed:
LPD-51911 - Add Sticker Dropdown for CMS

marcoscv-work committed:
LPD-55626 - Improve CMS Control Menu Styles and Apply CMS Theme

kevenleone committed:
LPD-56672 Create MarketplaceOAuth2 and Dynamic GraphQL filter to retrieve products for metrics

kevenleone committed:
LPD-56672 Redesign the useKPI hook to use GraphQL

kevenleone committed:
LPD-56672 Update Types

gabrielwas committed:
LPD-44846 Increment and simplify tests

gabrielwas committed:
LPD-44846 Not needed

kevenleone committed:
LPD-56672 Simplify

vamshikrishnamaddi committed:
LPD-55865 Add test

victorg1991 committed:
LPD-56487 Extract to be reused

victorg1991 committed:
LPD-56487 Create struts action for regenerating the structure

victorg1991 committed:
LPD-56487 Inject URL

marcogalluzzi committed:
LPD-54424 Simplify formatActionURL and add test

sandrodw3 committed:
LPD-56489 Simplify

gabrielwas committed:
LPD-44846 Id field is indexed

marcogalluzzi committed:
LPD-54424 Fix SimpleActionLinkRenderer and add test

sandrodw3 committed:
LPD-56489 Improve markup

gabrielwas committed:
LPD-44846 baseline

georgel-pop-lr committed:
LPD-56883 Improve permission logic to view marketplace, maintain the logic if there is INSTALL PAID APPS or INSTALL FREE APPS then automatically you have VIEW APPS permission

georgel-pop-lr committed:
LPD-56883 Move FeatureFlag out of permission logic method fragmentDisplayContext.isShowMarketplace, keeps code cleaner when this would be removed.

georgel-pop-lr committed:
LPD-56883 Don't allow user without permission to install fragment components from the marketplace, hide install button

georgel-pop-lr committed:
LPD-56883 Move the successful installing message before reloading the page so the user is able to see it.

sandrodw3 committed:
LPD-56489 Decrease z-index

sandrodw3 committed:
LPD-56489 Add new dropdown for CMS to regenerate the display page

gabrielwas committed:
LPD-44846 Fix existing tests

veroglez committed:
LPD-55891 Add ConfigContributor for the CKEditor 5 rich text fragment

sandrodw3 committed:
LPD-56489 Add new playwright test

gabrielwas committed:
LPD-44846 Create default label for new system objects

kevenleone committed:
LPD-54420 Add fallback to API requests

kevenleone committed:
LPD-54420 Simplify

rtienpal committed:
LPD-56395 Change class label for expire status

kevenleone committed:
LPD-54420 SF

gabrielwas committed:
LPD-44846 Unmodifiable objects can't have schedule fiedls and move feature flag checking to the resource layer

Commit:
LPD-56590 integration test

Commit:
LPD-56673 Adapt test to assert that multiselect picklist does not flicker when the user interacts with a mandatory field

jkappler committed:
LPD-52618 New role constant for CMS Administrator

This reverts commit 8cb73d747cbe4948ef7334679c1be4a087d16134.

jkappler committed:
LPD-52618 New util class that handles creating the CMS admin role and assigning some of its permissions

This reverts commit a5d22c7a0bc175b046e31191cffd08db16214d32.

jkappler committed:
LPD-52618 When an object entry folder is created assign the proper permissions

This reverts commit d4c5ad65cd5be7885180d6a17715ca9095524f96.

jkappler committed:
LPD-52618 Create regular role with its proper permissions once the instance is registered

This reverts commit 3ce798f3f3b7bf49fd8b050bc76cdbf22731e78f.

jkappler committed:
LPD-52618 Adds view permission for cms admin role

This reverts commit 4151b771460305c029481cf1500967fb869d80dc.

jkappler committed:
LPD-52618 Guest shouldn't have permissions, just like with the other pages

This reverts commit 92614b515236b601da652fe195d58b0bb63ac04b.

jkappler committed:
LPD-52618 Adds test for ObjectEntryFolderModelListener

This reverts commit dc4092b9cf661f39deeb923762d509f446cad309.

jkappler committed:
LPD-52618 ResourcePermissionLocalServiceImpl#getAvailableResourcePermissionActionIds never returns null

This reverts commit 7ce9ff5726b7d9aeb6f7624a2a50397d4d049409.

jkappler committed:
LPD-52618 Inline

This reverts commit 92a2110bead049a686316740019f1c93224d4d5a.

jkappler committed:
LPD-52618 Sort

This reverts commit ffecfa52f39e8fc9e775f30f3903c11f19aa4bc6.

georgel-pop-lr committed:
LPD-56883 Adapt tests

jkappler committed:
LPD-56626 Adds permissions after the object definition is created

izaera committed:
LPD-55605 It's enough to use portal-ext.properties

I was fooled because I didn't notice that LPD-34594 must be enabled too for
FDS cell renderers to be available.

marcogalluzzi committed:
LPD-54424 Fix

veroglez committed:
LPD-55891 Add hidden input when the rich text field is unlocalizable

marcoscv-work committed:
LPD-43525 - Copyright Date Update

marcoscv-work committed:
LPD-43525 - Add lfrbuild-releng-ignore

veroglez committed:
LPD-55891 Add test to check that the object entry for rich text field is saved correctly

thiagobuarqque committed:
LPD-56572 ignore third-party apache bundles that deploy configurations with non fully qualified class name as ID

rebsilva committed:
LPD-55897 Update test 'can access object entry via friendly URL' to change the separator friendly url

Commit:
LPD-56673 Remove loading/setLoading logic from MultipleSelection. Because no specific logic was left in MultipleSelection, I have moved handleAsyncOptions to base component and called it directly.

gabrielwas committed:
LPD-44846 buildService

izaera committed:
LPD-56904 Change test to reflect spec

Ids must be strings (that's what they were before the refactor) not
numbers.

This is probably because numbers in JavaScript are floats and may fail to
represent a big number coming from the Java side so better be safe and
treat ids as strings.

izaera committed:
LPD-56904 Apply fix

izaera committed:
LPD-56273 Create test for the bug

Actually I'm creating two tests: one for this LPD and another one to make
sure we don't break https://liferay.atlassian.net/browse/LPS-147618 with
this fix.

izaera committed:
LPD-56273 Fix the bug

izaera committed:
LPD-56273 SF

marcogalluzzi committed:
LPD-54424 Restore download folder option (LPD-50969) and apply it when navigating through folders

jkappler committed:
LPD-52618 Baseline

veroglez committed:
LPD-55891 Baseline

smottal committed:
LPD-55587 Rename for the new pattern

rafaellajordao committed:
LPD-56895 Mock groupId

rafaellajordao committed:
LPD-56895 Add and configure analytics-client-js

rafaellajordao committed:
LPD-56895 Add 'Verify events after navigating by SPA' test

georgel-pop-lr committed:
LPD-56883 Display console error only on development environment

rebsilva committed:
LPD-55897 Create unit test

georgel-pop-lr committed:
LPD-56883 Simplify get only one placedOrderItem

georgel-pop-lr committed:
LPD-56883 Move windows close on finally so it gets close on any kind of error

izaera committed:
LPD-55605 Remove non existent FF from Playwright tests

It was a leftover from previous commits but the FF no longer exists.

As per
https://github.com/brianchandotcom/liferay-portal/pull/162316#issuecomment-2934869761

juanjofgliferay committed:
LPD-57062 - Refactor. Add extra click to ensure publishing

brianchandotcom committed:
LPD-55891 Sort

brianchandotcom committed:
LPD-55891 Not needed

brianchandotcom committed:
LPD-55891 SF

brianchandotcom committed:
LPD-52618 Shouldn't this be an error?

marcogalluzzi committed:
LPD-54424 Fix test

brianchandotcom committed:
LPD-56489 SF

brianchandotcom committed:
LPD-44846 Sort

brianchandotcom committed:
LPD-44846 SF

brianchandotcom committed:
LPD-44846 modules/apps/object/object-service/src/main/resources/META-INF/sql/tables.sql

marcoscv-work committed:
LPD-43525 - Use Relative Path

dsitu committed:
LPD-56462 Remove poshi test

anthony-chu committed:
LPD-56255 Add method to determine if liferay version uses Jakarta

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPD-56883 Sort

pyoo47 committed:
LRCI-5978 Add linux/arm64 support

wanderlast committed:
LPD-57209 - set ERC based on parameter

anthony-chu committed:
LPD-56255 Rename

anthony-chu committed:
LPD-56255 Add test

anthony-chu committed:
LPD-56255 Group assertions together

pyoo47 committed:
LRCI-5978 Fix os/architecture detection logic

amosfong committed:
LRSD-9173 fix URLs with params

marcoscv-work committed:
LPD-55626 - Sort

wanderlast committed:
LPD-52709 - add test

gabrielwas committed:
LPD-57212 Fix BundleSiteInitializerTest tests

brianchandotcom committed:
LPD-56273 Sort

brianchandotcom committed:
LPD-56273 Sort

jkappler committed:
Revert "LPD-56626 Add property to definition so as to test it correctly"

This reverts commit 6feabd637ca12b5f35d79d9ad31c8e7c0b93bc66.

jkappler committed:
Revert "LPD-56626 We also have to replace the definition key with the current structure key"

This reverts commit 7c4d1e133cd6706ee75c34efd80a5d69d4e384b7.

liferay-translation committed:
LPS-77699 Update Translations

michelevigilante committed:
LPD-57261 Add a new suite for only commerce playwright tests

brianchandotcom committed:
LPD-43525 prep next

brianchandotcom committed:
LPD-43525 prep next

smottal committed:
LPD-57241 Fix semantic versioning

ambrinchaudhary committed:
LPD-57143 Test not needed, this validation was removed in LPD-56362

gianmarcobrunialti committed:
LPD-57248 Enable Sign In button for guest checkout too

rebsilva committed:
LPD-55897 buildLang

4lejandrito committed:
LPD-56260 This now needs to be duplicated or the test will fail

CalebJamesH committed:
LPD-56762 fetch all product specification on PDP

drewbrokke committed:
LPD-56255 project-templates-extensions: SF: buy space

marcogalluzzi committed:
LPD-57278 Fix - using straight single quotation mark

drewbrokke committed:
LPD-56255 project-templates-extensions: test: tests all test cases with the new method

brianchandotcom committed:
LRCI-5978 SF

brianchandotcom committed:
LPD-57209 Inline

brianchandotcom committed:
LPD-54596 Wordsmith

brianchandotcom committed:
LPD-54596 Regen

brianchandotcom committed:
LPD-56590 SF

brianchandotcom committed:
LPD-56590 Rename

brianchandotcom committed:
LPD-56590 auto SF

brianchandotcom committed:
LPD-55865 As used

brianchandotcom committed:
LPD-55865 Match the other test method names

brianchandotcom committed:
LPD-56165 SF

brianchandotcom committed:
LPD-56255 prep next

briangreenwald committed:
LPD-56243 Add JakartaCompatabilityUtil and test
